client/marketing/components/product-icon/icons/index.js

```diff
@@ -0,0 +1,24 @@
+export { default as blank } from './library/blank';
+
+// Amazon & Ebay Integration
+export { default as amazonEbayIntegration } from './library/amazon-ebay';
+export { default as woocommerceAmazonEbayIntegration } from './library/amazon-ebay';
+// AutomateWoo
+export { default as automatewoo } from './library/automatewoo';
+export { default as automatewooAlt } from './library/automatewoo-alt';
+// Facebook
+export { default as facebook } from './library/facebook';
+export { default as facebookForWoocommerce } from './library/facebook';
+// Google Ads
+export { default as googleAds } from './library/google';
+export { default as klikenMarketingForGoogle } from './library/google';
+// Hubspot
+export { default as hubspotForWoocommerce } from './library/hubspot';
+// Mailchimp
+export { default as mailchimpForWoocommerce } from './library/mailchimp';
+// Coupons
+export { default as woocommerceFreeGiftCoupons } from './library/gift';
+export { default as woocommerceGroupCoupons } from './library/people';
+export { default as woocommerceSmartCoupons } from './library/bulb';
+export { default as woocommerceStoreCredit } from './library/dollar';
```

Comment:
Thanks @danielbitzer had no idea - I thought they were custom. I'm not sure how to reconcile this now

Reply:
And I had no idea there was already an icon library available for G2 icons! 😮  Some are custom, but some are from the existing G2 icon Figma file. I don't seem able to find `@wordpress/icons` - could someone add / share it with me?

Reply:
@j111q you can view the icons package in Storybook https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library

> I'm not sure how to reconcile this now

@jconroy I think you can change the import so the icon is coming from `@wordpress/icons`. Probably also need to check backwards compatibility with the lowest supported WP version.

Reply:
Thanks I’ve already updated the imports @daniebitzer - Any pointers on the supported version stuff? I’d assumed it would just include them with this version

Reply:
So WC Admin's minimum supported WP version is 5.3 which means the min [Gutenberg version is 6.5](https://make.wordpress.org/core/2019/10/17/wordpress-5-3-field-guide/). And it looks like the icons package was added in [Gutenberg 7.4](https://github.com/WordPress/gutenberg/blob/master/changelog.txt#L1764-L1767) 😭